flexget/plugins/output/download.py

```diff
@@ -232,8 +232,13 @@ def download_entry(self, task, entry, url, tmp_path):
             auth = entry['download_auth']
             log.debug('Custom auth enabled for %s download: %s', entry['title'], entry['download_auth'])
 
+        headers = None
+        if 'download_headers' in entry:
+            headers = entry['download_headers']
+            log.debug('Custom headers enabled for %s download: %s', entry['title'], entry['download_headers'])
+
         try:
-            response = task.requests.get(url, auth=auth, raise_status=False)
+            response = task.requests.get(url, auth=auth, raise_status=False, headers=headers)
```

Comment:
No it doesn't override the custom headers. The default value of the headers parameter in the requests library is actually None so it has no affect. But I changed it to retrieve the default headers from the config anyway.

Reply:
Yes but if you pass in headers=None to the get request it will not override the existing headers. The headers passed in are merged with any existing headers.

Reply:
Thanks for all the patience with the PR btw :)

Reply:
I see.